server/workflow/workflow_test.go

```diff
@@ -159,4 +160,14 @@ func TestList(t *testing.T) {
 	rsp, err = bbClient.GetWorkflows(ctx2, req)
 	assert.Nil(t, err)
 	assert.Equal(t, 1, len(rsp.GetWorkflow()), "One workflow owned by USER2 should be returned")
+
+	if len(rsp.GetWorkflow()) == 1 {
```

Comment:
remove if? since you assert it above?

Reply:
`assert` isn't fatal, it just adds to a list of successes / failures that are reported at the end of the test.

There's `require` which is fatal but I'd prefer using `assert` whenever possible